packages/sources/src/components/application-heading/application-heading.models.ts

```diff
@@ -1,5 +1,5 @@
 export interface ApplicationHeading {
-  title: string;
+  title?: string;
```

Comment:
🤔 Dit ziet er raar uit. De title lijkt mij juist het fundament van "ApplicationHeading". Waarom is dit?

Reply:
@MichielDHVD Dit laten we nu, maar klinkt heel raar. Laten we dit oppakken in een nieuw (research) issue

Reply:
#1872